packages/testlab/src/skip-travis.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/testlab
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// tslint:disable-next-line:no-any
+export type TestCallbackRetval = void | PromiseLike<any>;
+
+export function itSkippedOnTravis(
```

Comment:
Please add a TSDOC comment to explain `testlab` users how to use this function and when should they use it (what kind of tests need to be skipped on Travis).